x-pack/qa/security-example-spi-extension/build.gradle

```diff
@@ -13,12 +14,16 @@ dependencies {
 }
 
 
-integTestRunner {
-    systemProperty 'tests.security.manager', 'false'
```

Comment:
Is this no longer needed?

Reply:
yes, tests pass without it so we probably don't want to have it in an example.